fruit-app/public/index.html

```diff
@@ -1,38 +1,58 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Fruit List Application</title>
-    <link rel="stylesheet" href="/pure-min-1.0.1.css" crossorigin="anonymous">
-    <link rel="stylesheet" href="/style.css" crossorigin="anonymous">
-  </head>
-  <body>
-    <div class="pure-g">
-      <div class="pure-u-1-3"></div>
-      <div class="pure-u-1-3">
-        <h1 style="text-align: center;">Fruit List Application</h1>
-
-        <form id="fruitForm" class="pure-form pure-g">
-          <div class="pure-u-3-4">
-            <input id="fruitInput" class="pure-input-1" placeholder="Pineapple...">
-          </div>
-          <div class="pure-u-1-4">
-            <button type="submit" class="pure-button pure-button-primary pure-input-1">Create</button>
-          </div>
-        </form>
-
-        <table style="margin: auto; width: 100%;" class="pure-table">
-          <thead>
-            <tr>
-              <th>ID</th>
-              <th>Name</th>
-              <th>Actions</th>
-            </tr>
-          </thead>
-          <tbody id="fruitList"></tbody>
-        </table>
-      </div>
-      <div class="pure-u-1-3"></div>
-    </div>
-    <script async src="/client.js"></script>
-  </body>
+
+<head>
+  <title>Fruit List Application</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="/app.css" crossorigin="anonymous">
+  <link rel="stylesheet" href="/patternfly.css" crossorigin="anonymous">
+  <link rel="stylesheet" href="/patternfly-addons.css" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body class="pf-u-display-flex pf-u-justify-content-center">
+  <div class="pf-l-stack pf-m-gutter" style="margin: auto; padding: 1em; min-width: 75%;">
+    <section class="pf-l-stack__item">
+      <h1 class="pf-c-title pf-m-4xl" style="text-align: center;">
+        <i class="fa fa-shopping-basket"></i>
+        Fruit List Application
+      </h1>
+    </section>
+
+    <section class="pf-l-stack__item">
+      <form id="fruitForm" novalidate class="pf-c-form">
+        <div class="pf-c-input-group">
+          <input class="pf-c-form-control" required id="fruitInput" class="pf-c-form"
+            placeholder="Pineapple..."></input>
+          <button class="pf-c-button pf-m-primary" type="submit">
+            Create
+          </button>
+        </div>
+      </form>
+    </section>
+
+    <section class="pf-l-stack__item pf-m-fill">
+      <table class="pf-c-table pf-m-grid-md" role="grid" id="frutiTable">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody id="fruitList"></tbody>
+      </table>
+    </section>
+  </div>
+
+  <script async src="/client.js"></script>
+</body>
+
+<footer>
```

Comment:
Is this in the latest build of the container, doesn't seem to be showing up for me?

Reply:
Yup, should be the latest compiled clients js code. Changes for adding a container around the body was done though css so should be reflected without a need for a recompile :thinking: 
Are you hosting it locally or through a openshift 4 cluster?

Reply:
Hosting it locally, I just pulled down your image for a look.

Reply:
Oh okay, could be due to the image wasn't the latest on quay.io. I just pushed the image if you want to pull the latest image and try again (or use the master tag since I set up build trigger for this branch, so should be the based on the latest commit).

Reply:
Yeah I built and ran it locally, so going to resolve this :+1: